docs/source/visualisation/experiment_tracking.md

```diff
@@ -1,4 +1,4 @@
-# Set up experiment tracking
+# Experiment tracking in Kedro-Viz
```

Comment:
I've updated the wording here to be consistent with the other titles of the sections under `Visualisation with Kedro-Viz`.